### PR TITLE
Add page for DMR

### PR DESCRIPTION
**Pull request by thrillstone**
Add a page for DMR: https://sol-jira.atlassian.net/browse/DATAGO-4013

### CODE REVIEW

**Comment by thrillstone** [comment]
image

File: docs/group_concepts/gc_dmr.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Dynamic Message Routing in PubSub+ Cloud
+=======================================
+
+In PubSub+ Cloud, all enterprise messaging services are created with the Dynamic Message Routing (DMR) feature enabled. To learn more about DMR,
+checkout `the official documentation in the Solace Tech Docs <https://docs.solace.com/Features/Dynamic-Msg-Routing.htm>`_.
```
Comment (mdspielman) [inline docs/group_concepts/gc_dmr.rst]: Should be check out.

File: docs/group_concepts/gc_dmr.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+PubSub+ Cloud will only prepare services for DMR in the following plans:
```
Comment (mdspielman) [inline docs/group_concepts/gc_dmr.rst]:
```suggestion
The following PubSub+ Cloud plans are DMR enabled:
```

File: docs/group_concepts/gc_dmr.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Kilo
+- Mega
+- Giga
+- Tera
+
+The message VPN created by PubSub+ Cloud has DMR enabled. A DMR cluster is automatically configured in the message VPN for the service. The password
```
Comment (mdspielman) [inline docs/group_concepts/gc_dmr.rst]:
```suggestion
The messaging service created by PubSub+ Cloud is DMR enabled. A DMR cluster is automatically configured on the PubSub+ Broker and enabled within the message VPN. The password
```

File: docs/group_concepts/gc_dmr.rst [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+for cluster communication is stored in PubSub+ Cloud and available from the status tab on the service details page, along with all other relevant DMR cluster
```
Comment (mdspielman) [inline docs/group_concepts/gc_dmr.rst]:
```suggestion
for cluster communication is stored in PubSub+ Cloud and available from the **Status** tab on the service details page, along with all other relevant DMR cluster
```

File: docs/group_concepts/gc_dmr.rst [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+information.
+
+Note that in PubSub+ Cloud, only external links can be created for DMR clusters at this time. To create DMR cluster links, use the
```
Comment (mdspielman) [inline docs/group_concepts/gc_dmr.rst]:
```suggestion
Note that in PubSub+ Cloud, you should only use external links to create event meshes. Internal links will be managed for you by PubSub+ Cloud. To create DMR cluster links, use the 
```

**Pull request merged**